collection/collection/build.gradle

```diff
@@ -58,92 +58,11 @@ kotlin {
     mingwX64()
 
     sourceSets {
-        all {
-            languageSettings.optIn("kotlin.RequiresOptIn")
-            languageSettings.optIn("kotlin.contracts.ExperimentalContracts")
-        }
-
         commonMain {
             dependencies {
-                api(libs.kotlinStdlib)
-                implementation("org.jetbrains.compose.annotation-internal:annotation:1.7.1")
-                implementation(libs.atomicFu)
-            }
-        }
-
-        commonTest {
-            dependencies {
-                implementation(libs.kotlinTest)
-                implementation(libs.kotlinTestAnnotationsCommon)
-                implementation(libs.kotlinCoroutinesCore)
+                api("androidx.collection:collection:1.5.0-beta02")
```

Comment:
Please use a version from catalog

Reply:
It is a pinned version, and we currently don't have pinned version in version catalogs.

I don't mind to have them there, but we have to explicitly decide to do that as a general approach.

If you talk about `libraryversions.toml` - we can't use it, can elaborate more if needed.

Reply:
Also, when we depend on not Stable version, we should create a task to replace it to Stable with Target version: Compose 1.8

Reply:
https://youtrack.jetbrains.com/issue/CMP-7495/Update-pinned-dependencies-before-navigation-stable-release